.github/workflows/frontend-e2e-test.yml

```diff
@@ -49,8 +49,9 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 16.x
-          cache: 'yarn'
-          cache-dependency-path: web/yarn.lock
+
+      - name: Install PNPM
```

Comment:
Why not use the officially provided
https://pnpm.io/continuous-integration#github-actions

Reply:
"When downloading the relevant environment, it seems that there are some restrictions. Maybe it can be solved by storing it in the warehouse through some link, but I don‘t find it. This picture is the reason why the three people are useless. Action is prohibited. It clearly meets the requirements he said, but it can't be used."`
![1661525327899](https://user-images.githubusercontent.com/72308243/186932473-8b98140c-2210-40ef-a2f0-b0967b946ae2.jpg)
![1661525339301](https://user-images.githubusercontent.com/72308243/186932506-24ccbb90-3d4b-45cc-ab04-5347a8ecb2fd.jpg)

@guoqqqi  You can look at the pictures. If anyone can solve it, it's best.

Reply:
OK, it seems to be due to some permissions of the Apache repository.🤣